package exceptionHandling;

public class ExceptionClass1 {
	
	public static Functions fun;
	
	public static void main(String[] args) {
		
		//1st Exception
		/*int a=10;
		int b=0;
		System.out.println(a/b);*/
		
		//2nd Exception
		/*String[] str= new String[2];		
		str[2]="Anand";*/
		
		//3rd Exception
		
		fun.fun1();

	}

}
